1-exercises/A-this/exercise-3.js

```diff
@@ -8,8 +8,13 @@
 
 let bankAccount = {
   currentAmount: 100,
-  deposit: function (amount) {},
-  withdraw: function (amount) {},
+  deposit: function (amount) {
+   return this.currentAmount = amount + this.currentAmount
+
+  },
+  withdraw: function (amount) {
+     return this.currentAmount = this.currentAmount - amount
```

Comment:
Excellent Kamran!
Just I want to tell you about a shorter version of this line.
like this = ```  this.currentAmount -= amount ```